compiler/ir.py

```diff
@@ -840,7 +841,7 @@ def apply_parallelization_to_node(
                 node_id, parallelizer, fileIdGen, fan_out
             )
         else:
-            raise Exception("Splitter not yet implemented")
+            raise UnparallelizableError("Splitter not yet implemented for command: {}".format(self.get_node(node_id=node_id).cmd_invocation_with_io_vars.cmd_name))
```

Comment:
Konstantinos (from old branch): Similarly, add which command the splitter doesn't exist for.